config/manager/rbac/role.yaml

```diff
@@ -6,19 +6,25 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
-- resources:
```

Comment:
This file is modified in 3 of the 4 patches. It is auto generated, correct? If so, it should be modified in only one patch, no?

Reply:
done, somehow missed it during rebasing and squashing